roles/container-engine/containerd/templates/config.toml.j2

```diff
@@ -1,78 +1,41 @@
-# persistent data location
-root = "{{ containerd_metadata_root_dir }}"
```

Comment:
Why did you remove root and state path configuration ? Not doable with v2 toml ?

Reply:
Yes, correct, This parameter is obsolete with Containerd CRI config v2.